docs/faq/Dump_query.md

```diff
@@ -0,0 +1,70 @@
+# QueryDump Interface Documentation
```

Comment:
1. needs to add this document in TOC.md then users can see this document in the content list